bigbluebutton-html5/imports/ui/components/actions-bar/component.jsx

```diff
@@ -43,10 +45,12 @@ export default class ActionsBar extends Component {
     );
   }
 
-  renderForUser() {
+  renderForUser(isVoiceUser) {
```

Comment:
i assume separating the renders was just for future proofing so if additional controls are added or different views then whoever implements it won't have to split it. The 2 render methods could remain the same but perhaps the shared code could be re used?